.gitignore

```
@@ -0,0 +1,8 @@
+# dependencies
+/node_modules
+
+# debug
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+.idea
```

```suggestion
.idea

```

We should add documentation on our standards for formatting, but I gently suggest we terminate all files with a new line per unix standards

---

Thanks for the suggestion but I am going to leave it as it is. The deadline is approaching and we need this merged ASAP. 

I left a TODO comment so we can handle this issue on a future PR :)

---

LGTM